### PR TITLE
DDF-1318: Updated empty array aguments to be null

### PR DESCRIPTION
**Pull request by AzGoalie**
@brendan-hofmann
@brianfelix
@kguerriero will hero

[Review on Reviewable](https://reviewable.io/reviews/codice/ddf-admin/92)

### CODE REVIEW

**Comment by AzGoalie** [comment]
I have them split up for readability. If this is a problem i can change it.

File: core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/ConfigurationAdmin.java [hunk 1]
```diff
@@ -417,7 +418,10 @@ public void updateForLocation(String pid, String location,
         // sanity check to make sure no values are
         // null
         for (Entry<String, Object> curEntry : configurationTable.entrySet()) {
-            if (curEntry.getValue() == null) {
+            Object value = curEntry.getValue();
+            if (value instanceof JSONArray && ((JSONArray)value).size() == 0) {
```
Comment (brendan-hofmann) [inline core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/ConfigurationAdmin.java]: JSONArray extends ArrayList, so this whole if block could be replaced with CollectionUtils.isEmpty().

File: core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/ConfigurationAdmin.java [hunk 2]
```diff
@@ -417,7 +417,8 @@ public void updateForLocation(String pid, String location,
         // sanity check to make sure no values are
         // null
         for (Entry<String, Object> curEntry : configurationTable.entrySet()) {
-            if (curEntry.getValue() == null) {
+            Object value = curEntry.getValue();
+            if (value == null || (value instanceof ArrayList && CollectionUtils.sizeIsEmpty(value))) {
```
Comment (brendan-hofmann) [inline core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/ConfigurationAdmin.java:421]: :+1:

**Comment by kguerriero** [comment]
Verified all tests pass and it builds

**Comment by brianfelix** [comment]
:+1:

**Pull request closed without merging**